runtime/gc_base/GCExtensions.cpp

```diff
@@ -319,7 +319,7 @@ MM_GCExtensions::needScanStacksForContinuationObject(J9VMThread *vmThread, j9obj
 {
 	bool needScan = false;
 #if JAVA_SPEC_VERSION >= 19
-	ContinuationState volatile *continuationStatePtr = VM_ContinuationHelpers::getContinuationStateAddress(vmThread, objectPtr);
+	volatile ContinuationState *continuationStatePtr = VM_ContinuationHelpers::getContinuationStateAddress(vmThread, objectPtr);
```

Comment:
while I prefer the new style, the old style should be still valid with identical effects - so I'd minimize the change and revert to the old style